src/legacy/core_plugins/tagcloud/public/tag_cloud_vis_options.tsx

```diff
@@ -0,0 +1,85 @@
+/*
```

Comment:
Nit: Maybe use `tag_cloud_options.tsx` as name here to match the component name.

Reply:
That makes sense. 
Also the `components` folder created for the future react components